core/templates/components/skiplinks/skiplinks.twig

```diff
@@ -0,0 +1,9 @@
+{% set skipID = "skip-to-content-" ~ random() %}
+
+<header>
```

Comment:
Do we need a wrapper around this? Isn't the idea that this link be placed as the first element after the body tag? If that is so, I would recommend removing or changing the `<header>` tag as to not compete with other header elements right after the body tag.

Reply:
Thanks - this has been removed.